storage_media/__manifest__.py

```diff
@@ -21,6 +21,8 @@
         "security/ir_rule.xml",
         "security/ir.model.access.csv",
     ],
-    "demo": [],
+    "demo": [
+        "data/storage_type.xml",
```

Comment:
```suggestion
        "demo/storage_type.xml",
```

and move the file ;)